app/assets/stylesheets/custom.css.scss

```diff
@@ -208,28 +208,39 @@ img {
   min-width: 100%;
 
   h1 {
-    font-size: 6em;
     color: $share-yellow;
-    text-shadow: 4px 2px #033E58;
+    font-size: 6em;
     padding: 2em 2em 0;
     text-align: center;
+    text-shadow: 4px 2px $share-dark-blue;
   }
 }
 
 @media (max-width: 850px){
   .org-name {
+    font-size: 1.5em;
     text-align: center;
   }
 
   .org-header{
 
     h1 {
-      font-size: 4em;
+      font-size: 3.5em;
       padding: 2em 0.5em 0;
     }
   }
 }
 
+// Org details
+
+@media (max-width: 499px){
+
+  #org-details-photo img{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space